Yoast/wordpress-seo
# Reintroduce wp-sanitize dependency

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:
* Because we only support the last 2 versions of WP, we can remove the hacky fix for https://github.com/Yoast/wordpress-seo/issues/11706.
* For context, see https://github.com/Yoast/wordpress-seo/pull/11718

## Test instructions

This PR can be tested by following these steps:

* See https://github.com/Yoast/wordpress-seo/pull/11718. WP 4.8 doesn't need to be tested anymore.

## UI changes

## Quality assurance


Fixes #
